src/SME.SGP.Aplicacao/Comandos/ComandosUsuario.cs

```diff
@@ -265,6 +265,10 @@ public void Sair()
         public async Task<string> SolicitarRecuperacaoSenha(string login)
         {
             var usuario = repositorioUsuario.ObterPorCodigoRfLogin(null, login);
+            if (usuario.Perfis == null || !usuario.Perfis.Any())
```

Comment:
A verificação de nulidade do usuário deve vir antes da verificação da nulidade dos perfis, pois se o usuário for nulo ocorrerá erro de referencia de objeto